server/middleware/maintenance.js

```diff
@@ -1,11 +1,37 @@
-const logger = require('../../log')
+const moment = require('moment')
+const config = require('../../config')
+const { isNullOrEmpty } = require('../helpers/utilities')
 
-module.exports = (req, res) => {
-  logger.info('GET maintenance view')
-  res.render('pages/maintenance', {
-    employeeName: req.user.employeeName,
-    csrfToken: res.locals.csrfToken,
-    hmppsAuthMFAUser: req.hmppsAuthMFAUser,
-    authUrl: req.authUrl,
-  })
+module.exports = (
+  { user: { employeeName = '' } = { employeeName: '' }, hmppsAuthMFAUser = false, authUrl = '' },
+  res,
+  next,
+) => {
+  try {
+    const {
+      maintenance: { start, end },
+    } = config
```

Comment:
FTR, this is here because the mocked values can only be changed for tests if the values are requested every time. Its a fudge basically and this was the cleanest solution I could muster.